src/main/webapp/app/exercises/shared/feedback/feedback-suggestion-badge/feedback-suggestion-badge.component.ts

```diff
@@ -12,12 +12,18 @@ export class FeedbackSuggestionBadgeComponent {
     @Input()
     feedback: Feedback;
 
+    @Input()
+    useDefaultText = false;
```

Comment:
Explanation for reviewers: The "default" feedback suggestion badge text is just "Suggestion" and is shown for text exercises where there is no distinction between accepted/discarded suggestions.